docs/samples/browser-plugin-meetings/app.js

```diff
@@ -956,36 +955,6 @@ const localMedia = {
   }
 }
 
-function handleTrackUnpublished(track, mediaElement, debugString) {
```

Comment:
I guess the published state change handler will be added back once the "published-state-update" event is added to the meeting object?

Reply:
Done: https://github.com/webex/webex-js-sdk/pull/3052

Reply:
in #3052 I can see the handler is back but it only calls stop(). Do we need to also call `clearVideoResolutionCheckInterval()` when main video is unpublished and reset srcObject properties on the audio/video elements? or is that done sowhere else now?

Reply:
For remote video, `clearVideoResolutionCheckInterval()` is called after meeting.leave() [here](https://github.com/webex/webex-js-sdk/blob/stream-classes/docs/samples/browser-plugin-meetings/app.js#L558). srcObjects are cleared during cleanUpMedia().

For local video, as we will use existing audio/video/share streams in another meeting so we don't have to call either of above methods. I can see local video and audio stream's srcObject is set to undefined but I will remove it once my older PR for effects will be applied on the new code.

Reply:
> For remote video, clearVideoResolutionCheckInterval() is called after meeting.leave() [here](https://github.com/webex/webex-js-sdk/blob/stream-classes/docs/samples/browser-plugin-meetings/app.js#L558). srcObjects are cleared during cleanUpMedia().

I'm only talking about local steams here, publishing is only for local streams not remote

> For local video, as we will use existing audio/video/share streams in another meeting

sorry I'm confused: in PR#3052 code was added to call stop() when stream is unpublished. You can't use a stopped stream for another meeting or anything else later. Once it's stopped, it cannot be resumed. 
Yes, in general with the webrtc-core track or stream classes you can reuse the same track/stream later, but that's not how the sample app works on beta branch right now. I agree it would be good to change this and re-use them, but that's a separate piece of work, let's not do it now as part of wcme 3.0 transition, because that in itself is big enough already.

Reply:
Done in #3053